jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimQueryController.java

```diff
@@ -57,6 +57,14 @@ public TimQueryController(OdeProperties odeProperties) {
       }
 
       RSU queryTarget = (RSU) JsonUtils.fromJson(jsonString, RSU.class);
+      
+      if (queryTarget.getRsuUsername() == null || queryTarget.getRsuUsername().isEmpty()) {
```

Comment:
SHould this be `curRsu.getRsuUsername() != null && !curRsu.getRsuUsername().isEmpty()`